woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1973,6 +1973,26 @@ def test_accessor_drop(sample_df):
     assert to_pandas(multiple_list_df).equals(to_pandas(different_order_df))
 
 
+def test_accessor_drop_inplace(sample_df):
+    inplace_df = sample_df.copy()
+    inplace_df.ww.init()
+    schema_df = inplace_df.copy()
+
+    if isinstance(inplace_df, pd.DataFrame):
+        inplace_df.ww.drop(['is_registered'], inplace=True)
+        assert len(inplace_df.ww.columns) == (len(sample_df.columns) - 1)
+        assert 'is_registered' not in inplace_df.ww.columns
+        assert to_pandas(schema_df).drop('is_registered', axis='columns').equals(to_pandas(inplace_df))
+    elif dd and isinstance(inplace_df, dd.DataFrame):
+        error = 'Drop inplace not supported for Dask'
+        with pytest.raises(ValueError, match=error):
+            inplace_df.ww.drop(['is_registered'], inplace=True)
+    elif ks and isinstance(inplace_df, ks.DataFrame):
+        error = 'Drop inplace not supported for Koalas'
+        with pytest.raises(ValueError, match=error):
+            inplace_df.ww.drop(['is_registered'], inplace=True)
```

Comment:
Maybe we should refactor this to have the same flow as the rename inplace test. Make the if and elif conditions test the dask and koalas versions and the test the pandas stuff inside an `else` block.

Reply:
Done!